src/Controller/ActivityController.php

```diff
@@ -479,4 +486,25 @@ public function applyForActivity(Activity $activity, ActivityUserRepository $act
         $activityUserRepo->apply($activity, $applierUser);
         return new JsonResponse(['message' => 'Applied with success!'], Response::HTTP_OK);
     }
+
+    /**
+     *Apply for an Activity.
+     * @Rest\Get("/{id}/applicants")
+     * @param Activity $activity
+     * @param ActivityUserRepository $activityUserRepo
+     * @return JsonResponse
+     */
+    public function listOfApplicants(Activity $activity, ActivityUserRepository $activityUserRepo): JsonResponse
+    {
+        $applicants = $activityUserRepo->getApplicantsForActivity($activity);
+        $context = SerializationContext::create();
```

Comment:
You are sure you don't need a context? What info is returned about the applicants?